app/src/main/res/drawable/ic_save_black_24dp.xml

```diff
@@ -4,6 +4,6 @@
         android:viewportWidth="24.0"
         android:viewportHeight="24.0">
     <path
-        android:fillColor="#FFFFFFFF"
+        android:fillColor="#FF000000"
```

Comment:
It must be white

Reply:
Besides, I think, we must modify `accent` color inside `colors.xml`, because I think that it is better to make this button blue or, maybe, a little bit different with `primary` and `primary_dark`